pdr_backend/analytics/predictoor_dashboard/dash_components/view_elements.py

```diff
@@ -168,7 +168,11 @@ def get_table(table_id, table_name, searchable_field, columns, data):
                         value=True,
                     )
                     if table_name == "Feeds"
-                    else None
+                    else dbc.Switch(
+                        id="show-favourite-addresses",
+                        label="Toggle my predictoor feeds",
```

Comment:
Let's rename this to something more specific like: "Select my predictoors", "Select provided predictoors", "Select configured predictoors",  any other ideas?
I like the "Select configured predictoors" one.